packages/fluentui/e2e/tsconfig.json

```diff
@@ -3,9 +3,10 @@
   "compilerOptions": {
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27686&commitId=13df44e327bc5ccb14ef6d35b279a8b57c54b05d&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1258 screenshots unchanged, 0 screenshots with different dimensions and 2 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checkbox Example Checked Mixed.chromium.png|72| Changed|
|Checkbox.Checkbox Example.chromium.png|51| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>